txpool/txpool.go

```diff
@@ -548,13 +554,17 @@ func (p *TxPool) validateTx(tx *types.Transaction) error {
 	// Extract the sender
 	from, signerErr := p.signer.Sender(tx)
 	if signerErr != nil {
+		metrics.IncrCounter([]string{txPoolMetrics, "invalid_signature_txs"}, 1)
```

Comment:
IMO, we should unify all these invalid transactions under single metrics ("invalid_tx"). I would like to hear other opinions as well.

Reply:
I think it is more useful to keep it separated. That way you have a more precise measure what the users are sending, or possibly identify a malicious actor.

Reply:
That's correct, but it has a couple of downsides as well:
- you don't have a cumulative number which is suggesting how many invalid transactions in total are in some time period. Having multiple metrics is great if you are really interested in details, but I'm afraid the essence is lost and you can't see the forest for the trees.
- @praetoriansentry needs to create a DD dashboard that holds ~20 metrics, which I'm not sure he is keen doing :smile: 

So to me, let's discuss it and find the most appropriate solution. Mitigation would be to provide cumulative metrics (e.g. `invalid_txs`), that would report all the invalid transactions and then we have the best out of both worlds.

Reply:
Ok, agreed. My understanding is we want it by type of error, but I am maybe wrong.

Reply:
I don't find necessary to provide cumulative metrics, that's easily done in the metrics UI, DD, Grafana, etc.